### DISCUSSION TITLE
ne marche plus

### DISCUSSION THREAD

**Discussion by callofludo** [MartinPELCAT/GlobalExamFinisher#3] [General]
impossible de mettre le code. il ne lance rien

**Comment by callofludo** [comment]
ca marche c'est bon mais est-il possible d'avoir le meme script mais seulement pour faire 1 heure ?

**Reply by MartinPELCAT** [reply]
Change le 40 par un autre nombre dans la boucle for

```
const buttons = document.querySelectorAll("a.button-solid-primary-small");
for (let index = 0; index < 15; index++) {
  setTimeout((idx) => {
    window.open(buttons[idx].href);
  }, index * 1000, index);
}

```

**Comment by p1001a** [comment]
Hello ! 
ça ne fonctionne plus... Vous auriez autre chose svp ? 
Sauve une classe ! 😭
